src/styles/c-navigation.scss

```diff
@@ -78,8 +84,11 @@ ul {
 .navbar {
   display: block;
   padding: 0;
-  transition: margin 250ms ease;
+  transition: all 250ms ease;
   order: -1; /* This is needed for ie - due to it beeing render after slots */
+  height: calc(75vh - 64px);

```

Comment:
Would be nice with a comment describing what the this calculation is about

Reply:
comment has been added in the new changes